modules/ROOT/pages/troubleshoot-service-binding-issues.adoc

```diff
@@ -52,3 +53,37 @@ The status of binding displays as `ReferencesNonexistentInstance`.
 
 Delete the erroneous binding command and re-create it with the correct input.
 
+
+[[error-instance-not-ready]]
+== Error instance not ready
+
+When you create a binding, the referenced API or adapter must be ready. Otherwise, you receive the 'ErrorInstanceNotReady' error.
+
+== Causes
```

Comment:
I think you missed one level

```suggestion
=== Causes
```

Reply:
ha, copied and pasted from another place, you are right!